File: 23-tagging/npm-package/tagging.ts
import axios from 'axios';
const API = 'https://api.costkatana.com/api'\;
const KEY = process.env.COST_KATANA_API_KEY;

export async function addTags(requestId: string, tags: string[]) {
  const res = await axios.post(`${API}/tags`, {requestId, tags},
    {headers: {'Authorization': `Bearer ${KEY}`}});
  console.log('✅ Tags added');
  return res.data.data;
}

File: 23-tagging/python-sdk/tagging.py
import os, requests
API = "https://cost-katana-backend.store/api"
KEY = os.getenv("COST_KATANA_API_KEY")

def add_tags(request_id, tags):
    res = requests.post(f"{API}/tags",
        json={"requestId": request_id, "tags": tags},
        headers={"Authorization": f"Bearer {KEY}"})
    res.raise_for_status()
    print("✅ Tags added")
    return res.json()["data"]
